### PR TITLE
Fix #606 - Prevent database hosts bulk selection if host has any databases

### LINKED ISSUE #606: [500s] Using "Bulk Delete" on Database Hosts Results in 500
Trying to use the Bulk Delete option on the List Database Hosts, you get a 500. Ideally, this shouldn't be checkable if the database host has databases. There is logic on the Edit Database Host page, but not on the List.

### PR DESCRIPTION
**Pull request by lancepioch**
Fixes #606

### CODE REVIEW

File: app/Filament/Resources/DatabaseHostResource/Pages/ListDatabaseHosts.php [hunk 1]
```diff
@@ -31,14 +31,17 @@ public function table(Table $table): Table
                 TextColumn::make('port')
                     ->sortable(),
                 TextColumn::make('username')
+                    ->icon('tabler-user')
                     ->searchable(),
-                TextColumn::make('max_databases')
-                    ->numeric()
-                    ->sortable(),
```
Comment (Boy132) [inline app/Filament/Resources/DatabaseHostResource/Pages/ListDatabaseHosts.php:37]: Why was this removed?

File: app/Filament/Resources/DatabaseHostResource/Pages/ListDatabaseHosts.php [hunk 2]
```diff
@@ -31,14 +31,17 @@ public function table(Table $table): Table
...
+                TextColumn::make('databases_count')
+                    ->counts('databases')
+                    ->icon('tabler-database')
+                    ->label('Databases'),
                 TextColumn::make('node.name')
-                    ->numeric()
+                    ->icon('tabler-server-2')
                     ->sortable(),
```
Comment (Boy132) [inline app/Filament/Resources/DatabaseHostResource/Pages/ListDatabaseHosts.php:42]: Why should add a placeholder here so the column isn't empty when no node is linked.

File: app/Filament/Resources/DatabaseHostResource/Pages/ListDatabaseHosts.php [hunk 3]
```diff
@@ -31,14 +31,17 @@ public function table(Table $table): Table
                 TextColumn::make('port')
                     ->sortable(),
                 TextColumn::make('username')
+                    ->icon('tabler-user')
```
Comment (Boy132) [inline app/Filament/Resources/DatabaseHostResource/Pages/ListDatabaseHosts.php:34]: Wouldn't adding a user icon suggest that it's a panel user?

**Review by Boy132 — CHANGES_REQUESTED** [review]

File: app/Filament/Resources/DatabaseHostResource/Pages/ListDatabaseHosts.php (hunk 3 above)
Comment (lancepioch) [inline app/Filament/Resources/DatabaseHostResource/Pages/ListDatabaseHosts.php:34]:
```suggestion
```

Good point

Comment (lancepioch) [reply app/Filament/Resources/DatabaseHostResource/Pages/ListDatabaseHosts.php]: Good point

File: app/Filament/Resources/DatabaseHostResource/Pages/ListDatabaseHosts.php (hunk 1 above)
Comment (lancepioch) [reply app/Filament/Resources/DatabaseHostResource/Pages/ListDatabaseHosts.php:37]: "Max" limit for a resource isn't a good metric to display on the overview. You display current metrics instead.

File: app/Filament/Resources/DatabaseHostResource/Pages/ListDatabaseHosts.php (hunk 2 above)
Comment (lancepioch) [reply app/Filament/Resources/DatabaseHostResource/Pages/ListDatabaseHosts.php:41]: Good idea!

Comment (lancepioch) [reply app/Filament/Resources/DatabaseHostResource/Pages/ListDatabaseHosts.php:41]: I tried it out and I don't think it looks better.

image

image

Comment (Boy132) [reply app/Filament/Resources/DatabaseHostResource/Pages/ListDatabaseHosts.php:41]: Did you use `->placeholder`?

**Review by Boy132 — APPROVED** [review]
